### ISSUE TITLE
Unable to build on Windows

### ISSUE THREAD

**Issue by weihongyu12** [nuxt-modules/html-validator#232]
### Version
module: @nuxtjs/html-validator@1.1.0
nuxt: 3.0.0-rc.12

#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [x] universal
 - [ ] spa

## Additional information

```
- Operating System: `Windows_NT`
- Node Version:     `v16.18.0`
- Nuxt Version:     `3.0.0-rc.12`
- Nitro Version:    `0.6.0`
- Package Manager:  `yarn@1.22.19`
- Builder:          `webpack`
- User Config:      `srcDir`, `modules`, `head`, `imports`, `builder`, `build`, `hooks`, `webpack`
- Runtime Modules:  `@nuxtjs/tailwindcss@5.3.5`, `@vueuse/nuxt@9.3.1`, `@nuxtjs/html-validator@1.1.0`
- Build Modules:    `-`
```

## Checklist
* [x]  I have tested with the latest Nuxt version and the issue still occurs
* [x]  I have tested with the latest module version and the issue still occurs
* [x]  I have searched the issue tracker and this issue hasn't been reported yet

### Steps to reproduce

On the Windows, initialize the project, then install and enable the '@nuxtjs/html-validator' module, then run the project's build command.

```
yarn build
```

### What is expected?

If it is normal, it should be built successfully or HTML verification information should be given, and an exception should not be thrown.

### What is actually happening?

A build exception was thrown: `Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'`

This exception is only thrown on Windows. MacOS and Linux don't have this problem.

![image](https://user-images.githubusercontent.com/19792747/196838358-c2c68d0d-75d4-4533-b15d-8049182c9a2c.png)

**Comment by AndrewBogdanovTSS** [comment]
Getting the same issue

**Comment by danielroe** [comment]
Would you test with `v1.1.1`?

**Comment by weihongyu12** [comment]
Still the same problem, here is the new console output log：

![image](https://user-images.githubusercontent.com/19792747/199627467-4984599f-6baa-445b-85b5-d2e56c8bed4f.png)

**Comment by AndrewBogdanovTSS** [comment]
I confirm that issue is still present with `v.1.1.1`
![image](https://user-images.githubusercontent.com/2766448/199666118-0bb31494-6f6f-47f3-95f7-92eb5dffcbd9.png)

**Comment by ineshbose** [comment]
Facing this issue as well!

**Comment by weihongyu12** [comment]
https://github.com/nuxt/framework/issues/8461

**Comment by weihongyu12** [comment]
https://github.com/Baroshem/nuxt-security/issues/42

**Comment by danielroe** [comment]
I believe this is now fixed with the latest release of `ufo@1.0.1` via https://github.com/unjs/ufo/pull/106. 

Would you try regenerating your lockfile or running `npx nuxi upgrade` or `npm up ufo` or equivalent?

If you can still reproduce and your lockfile only contains ufo 1.0.1, please let me know with a new reproduction and I'll reopen.
